src/pages/home/library/MintSong.tsx

```diff
@@ -1,4 +1,4 @@
-import { Box, Button as MUIButton, Stack } from "@mui/material";
+import { AlertTitle, Box, Button as MUIButton, Stack } from "@mui/material";
```

Comment:
Inserting minor change as this is a small enough PR to update copy/colors to match mocks